libs/api/domains/ministry-of-justice/src/lib/models/SearchCaseTemplate.response.ts

```diff
@@ -0,0 +1,11 @@
+import { Field, Int, ObjectType } from '@nestjs/graphql'
+import { Case } from './Case.model'
+
+@ObjectType('MinistryOfJusticeSearchCaseTemplateResponse')
+export class SearchCaseTemplateResponse {
```

Comment:
You can use the `PagintedResponse` function from the `@island.is/nest/pagination` library to extend from which add the `totalCount` and `pageInfo` props.